docs/ota-client-guide/modules/ROOT/pages/customise-targets-metadata.adoc

```diff
@@ -0,0 +1,61 @@
+
+The Uptane keys we advise you to rotated offline are `root.json` and `targets.json` metadata files in the *Image Repository*. These are referred to as root-of-trust keys For more information on these metadata files please refer to xref:uptane.adoc#_uptane_metadata[Uptane metadata overview]. The `targets.json` metadata file for the Image repository is responsible for signing software. It has a `custom` field that can be edited to suit your needs or requirements. Below are a few scenarios where you might want to edit the `targets.json` file.
+
+
+
+== Possible Use Cases to customise `targets.json`
+
+A few cases might occur where you will have to edit the custom field within the `targets.json` file. These might include:
+
+* to install instructions or scripts for an ECU
+* to add extra tracking fields for internal compliance or auditing
+* to unlock codes
+* to add material for configuration
+* to add an extra notice or text to be displayed for your users
+
+
+[NOTE]
+====
+For more information on additional use cases which will require customisation of the `targets` metadata file, please refer to the section https://uptane.github.io/papers/ieee-isto-6100.1.0.0.uptane-standard.html#rfc.section.5.2.3.1.1['Custom metadata about images' in the uptane standard].
+====
+
+
+
+== Steps in updating `targets.json` metadata
+
+. Before editing the `targets.json` metadata file, make sure you already have your new `root` and `targets` metadata keys generated and signed by the old keys. To generate and rotate the new Uptane keys for `root` and `targets` metadata please refer to xref:rotating-signing-keys.adoc#_rotate_the_keys_for_root_and_targets_metadata[managing keys for software metadata].
+. With your new `root` and `targets` keys generated, open the `targets.json` file and navigate to the `custom` field as shown in the image below.
+
+.Example of a `targets.json` file indicating custom field in white box
+[caption="Figure 1: "]
+image::targets-custom-field.png[]
+[start=3]
+. Maintain the default values specified, however you can add any extra values you want within this `custom` field as shown in the image below.
+
+.Example of new values (in white box) added to custom field in `targets.json` file
+[caption="Figure 2: "]
+image::targets-custom-field-edited.png[]
+[start=4]
+. After editing the `targets.json` file, you need to sign the new file with the old `targets.json` metadata file. To do this, follow point 3 of xref:rotating-signing-keys.adoc#_rotate_the_keys_for_root_and_targets_metadata[managing keys for software metadata], and proceed with point 4 to upload the newly signed `targets.json` to OTA Connect.
++
+[IMPORTANT]
+====
+Each time the `targets.json` fie is edited, it must be signed by the former `targets` key and uploaded to OTA Connect.

```

Comment:
I think this is a little confusing, largely because it's not clear what you mean by "the former targets key".

```suggestion
Each time the `targets.json` file is edited, it must be signed by your offline `targets` key, and then uploaded to OTA Connect.

```